openassessment/fileupload/views_filesystem.py

```diff
@@ -17,8 +17,6 @@ def filesystem_storage(request, key):
     """
     Uploading and download files to the local filesystem backend.
     """
-    if isinstance(key, unicode):
-        key = key.encode("utf-8")
```

Comment:
Once we make the above two changes, we need to make this change too, to get `test_upload_download_with_accented_key()` to pass.